server/src/stepIngredient/stepIngredient.controller.js

```diff
@@ -0,0 +1,14 @@
+import { StepIngredient } from '@/shared/database/index';
+
+const newStepIngredient = async (req, res, next) => {
```

Comment:
I'm 99% sure we won't ever need a controller for adding step ingredients. This is an intermediary table meaning that it only has context in relation to step and ingredient and will be added as part of adding steps inside the `createStep` (or `newStep` as you called it). 

All that means that you can probably remove this file, `stepIngredients/index.js` and the route inside `/router.js`